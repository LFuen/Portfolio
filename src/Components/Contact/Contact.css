.margin {
    margin-top: 10px;
    margin-bottom: 100px;
}

h2 {
    color: whitesmoke;
    font-family: 'Roboto Mono', monospace;
}

div.bg-color {
    padding: 25px 10px 25px 10px;
}