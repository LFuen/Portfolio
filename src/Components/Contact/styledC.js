import {fadeInDown} from 'react-animations'
import styled, {keyframes} from 'styled-components'

const fadeIn = keyframes`${fadeInDown}`

const ContactDiv = styled.div`
    animation: 2s ${fadeIn};
`
export {
    ContactDiv
}