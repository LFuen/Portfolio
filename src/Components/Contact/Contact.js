import React, {Component} from 'react'
import './Contact.css'


class Contact extends Component{

    render(){
        return (
        <div className='App'>
            <article className='margin'>
                <h1 id='Contact'>Contact</h1>
                <div>
                    <p>You can find links to my profile(s) and E-Mail below:</p>
                    <div className='bg-color'>
                        <p>
                            <a   
                                href='https://www.linkedin.com/in/lilifuentes/' 
                                target='_blank' 
                                rel='noreferrer'>
                                <span title='Lili Fuentes on LinkedIn'>
                                    <img    
                                        className='regSize' 
                                        alt='LinkedIn' 
                                        src='https://lfuen.github.io/portimages/linkedIn-1.png'/>
                                </span>
                            </a>
                        </p>

                        <p>
                            <a 
                                href='https://www.github.com/LFuen' 
                                target='_blank' 
                                rel='noreferrer'>
                                <span title='Lili Fuentes on GitHub'>
                                    <img 
                                        className='regSize' 
                                        alt='GitHub' 
                                        src='https://lfuen.github.io/portimages/github-1.png'/>
                                </span>
                            </a>
                        </p>
                        
                        <p>
                            <a 
                                href='https://www.instagram.com/lilifuen' 
                                target='_blank' 
                                rel='noreferrer'>
                                <span title='Lili Fuentes on Instagram'>
                                    <img 
                                        className='regSize' 
                                        alt='Instagram' 
                                        src='https://lfuen.github.io/portimages/theGram-1.png'/>
                                </span>
                            </a>
                        </p>
                        
                        <p>
                            <a
                                href='mailto:lile7886@gmail.com' 
                                target='_blank' 
                                rel='noreferrer'>
                                <span title='LilE7886@GMail.com'>
                                    <img 
                                        className='regSize' 
                                        alt='E-Mail' 
                                        src='https://lfuen.github.io/portimages/email-1.png'/>
                                </span>
                            </a>
                        </p>
                    </div>
                </div>
            </article>
        </div>
        );
    }
} 


export default Contact