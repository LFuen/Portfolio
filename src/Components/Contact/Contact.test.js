import React from 'react'
import ReactDOM from 'react-dom'
import { BrowserRouter } from 'react-router-dom'
import renderer from 'react-test-renderer'
import Contact from './Contact'



describe(`Contact component`, () => {
    describe(`Snapshot`, () => {
        it(`renders the UI as expected`, () => {
            const contact = renderer.create(
            <BrowserRouter>
                <Contact />
            </BrowserRouter>).toJSON()
            expect(contact).toMatchSnapshot()
        })
    })

    describe(`Smoke test`, () => {
        it(`Renders without crashing`, () => {
            const div = document.createElement('div')

            ReactDOM.render(
                <BrowserRouter>
                    <Contact />
                </BrowserRouter>,
                div
            )
            ReactDOM.unmountComponentAtNode(div)
        })
    })
})