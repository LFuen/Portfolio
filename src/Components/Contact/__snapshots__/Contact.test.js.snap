// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact component Snapshot renders the UI as expected 1`] = `
<div
  className="App"
>
  <article
    className="margin"
  >
    <h1
      id="Contact"
    >
      Contact
    </h1>
    <div>
      <p>
        You can find links to my profile(s) and E-Mail below:
      </p>
      <div
        className="bg-color"
      >
        <p>
          <a
            href="https://www.linkedin.com/in/lilifuentes/"
            rel="noreferrer"
            target="_blank"
          >
            <span
              title="Lili Fuentes on LinkedIn"
            >
              <img
                alt="LinkedIn"
                className="regSize"
                src="https://lfuen.github.io/portimages/linkedIn-1.png"
              />
            </span>
          </a>
        </p>
        <p>
          <a
            href="https://www.github.com/LFuen"
            rel="noreferrer"
            target="_blank"
          >
            <span
              title="Lili Fuentes on GitHub"
            >
              <img
                alt="GitHub"
                className="regSize"
                src="https://lfuen.github.io/portimages/github-1.png"
              />
            </span>
          </a>
        </p>
        <p>
          <a
            href="https://www.instagram.com/lilifuen"
            rel="noreferrer"
            target="_blank"
          >
            <span
              title="Lili Fuentes on Instagram"
            >
              <img
                alt="Instagram"
                className="regSize"
                src="https://lfuen.github.io/portimages/theGram-1.png"
              />
            </span>
          </a>
        </p>
        <p>
          <a
            href="mailto:lile7886@gmail.com"
            rel="noreferrer"
            target="_blank"
          >
            <span
              title="LilE7886@GMail.com"
            >
              <img
                alt="E-Mail"
                className="regSize"
                src="https://lfuen.github.io/portimages/email-1.png"
              />
            </span>
          </a>
        </p>
      </div>
    </div>
  </article>
</div>
`;
